### PR TITLE
DW-5237 fixed certs issue on preprod, not sure how it previously work…

### PR DESCRIPTION
**Pull request by sandeepg-github**
…ed on lower envs

### CODE REVIEW

File: locals.tf [hunk 1]
```diff
@@ -58,19 +58,19 @@ locals {
...
-    development = "ucfs_ca"
-    qa          = "ucfs_ca"
-    integration = "ucfs_ca"
-    preprod     = "ucfs_ca"
-    production  = "ucfs_ca"
+    development = "dataworks_root_ca,dataworks_mgt_root_ca"
+    qa          = "dataworks_root_ca,dataworks_mgt_root_ca"
+    integration = "dataworks_root_ca,dataworks_mgt_root_ca,ucfs_ca"
+    preprod     = "dataworks_root_ca,dataworks_mgt_root_ca,ucfs_ca"
+    production  = "dataworks_root_ca,dataworks_mgt_root_ca,ucfs_ca,ucfs_ca_old"
```
Comment (benjaminsherwood) [inline locals.tf:65]: Doesn't need the ``ucfs_ca_old``

File: terraform.tf.j2 [hunk 2]
```diff
@@ -160,3 +173,18 @@ data "terraform_remote_state" "management_dns" {
...
+  config = {
+    bucket         = "{{terraform.state_file_bucket}}"
+    key            = "terraform/dataworks/aws-certificate-authority.tfstate"
+    region         = "{{terraform.state_file_region}}"
+    encrypt        = true
+    kms_key_id     = "arn:aws:kms:{{terraform.state_file_region}}:{{terraform.state_file_account}}:key/{{terraform.state_file_kms_key}}"
+    dynamodb_table = "remote_state_locks"
+
+  }
+}
```
Comment (benjaminsherwood) [inline terraform.tf.j2:190]: New line required pls :)

Comment (benjaminsherwood) [reply terraform.tf.j2:190]: Actually - please can you put this further up the file with the rest of the remote states.

**Review by benjaminsherwood — CHANGES_REQUESTED** [review]

File: terraform.tf.j2 (hunk 2 above)
Comment (sandeepg-github) [reply terraform.tf.j2]: done, it is actually just below the rest of the remote states.

File: locals.tf (hunk 1 above)
Comment (sandeepg-github) [reply locals.tf]: done

**Review by benjaminsherwood — APPROVED** [review]

**Pull request merged**